### DISCUSSION TITLE
What happened to version2024.06?

### DISCUSSION THREAD

**Discussion by mayrstefan** [uyuni-project/uyuni#8985] [General]
There was a tag for 2024.06 three weeks ago: https://github.com/uyuni-project/uyuni/tree/Uyuni-2024.06
But I don't see a regular release. What happened to it?

**Comment by deneb-alpha** [comment]
> There was a tag for 2024.06 three weeks ago: https://github.com/uyuni-project/uyuni/tree/Uyuni-2024.06 But I don't see a regular release. What happened to it?

Hi @mayrstefan, 
that tag will be moved and as usual we will crate the release branch. We had some turbulence  and the release wasn't good to go. We are going to release directly `2024.07` instead of a belated 2024.06.

**Reply by mayrstefan** [reply]
May we know what caused missing the june release?

**Reply by rjmateus** [reply]
We made some significant changes in the deployment process because of the containerized version, and we decided to give extra room to our Quality Engineering team to test all the installation and migration process.
Since we decided to add support for some more UYUNI versions based on RPM, it also reduced the urgency, and we prefer to bring more stability with an even more tested version.

**Comment by mayrstefan** [comment]
Thank you
